### PR TITLE
azure-devops: fix url connection string

### LINKED ISSUE #724: azure dev-ops git commit fails to connect
```
Initializing Logger with LogLevel: INFO
11-18-2022 16:22:35 INFO     Loading CommittimeTypeConfig, inputs below:
11-18-2022 16:22:35 INFO     provider='git', from env var PROVIDER
11-18-2022 16:22:35 INFO     Loading GitCommittimeConfig, inputs below:
11-18-2022 16:22:35 INFO     username='whayutin', from env var API_USER
11-18-2022 16:22:35 INFO     token=REDACTED, from env var TOKEN
11-18-2022 16:22:35 INFO     namespaces='mongo-persistent,basic-python-tekton', from env var NAMESPACES
11-18-2022 16:22:35 INFO     git_api='https://dev.azure.com/whayutin', from env var GIT_API
11-18-2022 16:22:35 INFO     git_provider='azure-devops', from env var GIT_PROVIDER
11-18-2022 16:22:35 INFO     tls_verify=True, default value; TLS_VERIFY was not set
11-18-2022 16:22:35 INFO     Watching namespaces: {'mongo-persistent', 'basic-python-tekton'}
/home/whayutin/OPENSHIFT/git/KONVEYOR/PELORUS/upstream/pelorus/.venv/lib64/python3.10/site-packages/urllib3/connectionpool.py:1045: InsecureRequestWarning: Unverified HTTPS request is being made to host 'api.cluster-wdh11092022b.wdh11092022b.mg.dog8code.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/1.26.x/advanced-usage.html#ssl-warnings
  warnings.warn(
11-18-2022 16:22:36 ERROR    Error encountered while getting CommitMetric info:
11-18-2022 16:22:36 ERROR    not all arguments converted during string formatting
Traceback (most recent call last):
  File "/home/whayutin/OPENSHIFT/git/KONVEYOR/PELORUS/upstream/pelorus/exporters/committime/collector_base.py", line 259, in get_metric_from_build
    metric = self._set_commit_timestamp(metric, errors)
  File "/home/whayutin/OPENSHIFT/git/KONVEYOR/PELORUS/upstream/pelorus/exporters/committime/collector_base.py", line 386, in _set_commit_timestamp
    metric = self.get_commit_time(metric)
  File "/home/whayutin/OPENSHIFT/git/KONVEYOR/PELORUS/upstream/pelorus/exporters/committime/collector_azure_devops.py", line 33, in get_commit_time
    logging.debug("metric.git_api %s" % (self.git_api))  # TODO: metric, not self
TypeError: not all arguments converted during string formatting
/home/whayutin/OPENSHIFT/git/KONVEYOR/PELORUS/upstream/pelorus/.venv/lib64/python3.10/site-packages/urllib3/connectionpool.py:1045: InsecureRequestWarning: Unverified HTTPS request is being made to host 'api.cluster-wdh11092022b.wdh11092022b.mg.dog8code.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/1.26.x/advanced-usage.html#ssl-warnings
  warnings.warn(
```

### PR DESCRIPTION
**Pull request by weshayutin**
## Linked Issues?

resolves #724 

w/ fix:
```
11-18-2022 17:07:17 INFO     Loading CommittimeTypeConfig, inputs below:
11-18-2022 17:07:17 INFO     provider='git', from env var PROVIDER
11-18-2022 17:07:17 INFO     Loading GitCommittimeConfig, inputs below:
11-18-2022 17:07:17 INFO     username='whayutin', from env var API_USER
11-18-2022 17:07:17 INFO     token=REDACTED, from env var TOKEN
11-18-2022 17:07:17 INFO     namespaces='mongo-persistent,basic-python-tekton', from env var NAMESPACES
11-18-2022 17:07:17 INFO     git_api='https://dev.azure.com/whayutin', from env var GIT_API
11-18-2022 17:07:17 INFO     git_provider='azure-devops', from env var GIT_PROVIDER
11-18-2022 17:07:17 INFO     tls_verify=True, default value; TLS_VERIFY was not set
11-18-2022 17:07:17 INFO     Watching namespaces: {'mongo-persistent', 'basic-python-tekton'}
/home/whayutin/OPENSHIFT/git/KONVEYOR/PELORUS/upstream/pelorus/.venv/lib64/python3.10/site-packages/urllib3/connectionpool.py:1045: InsecureRequestWarning: Unverified HTTPS request is being made to host 'api.cluster-wdh11092022b.wdh11092022b.mg.dog8code.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/1.26.x/advanced-usage.html#ssl-warnings
  warnings.warn(
11-18-2022 17:07:18 INFO     metric.commit_time 2022-11-11 19:59:51+00:00
/home/whayutin/OPENSHIFT/git/KONVEYOR/PELORUS/upstream/pelorus/.venv/lib64/python3.10/site-packages/urllib3/connectionpool.py:1045: InsecureRequestWarning: Unverified HTTPS request is being made to host 'api.cluster-wdh11092022b.wdh11092022b.mg.dog8code.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/1.26.x/advanced-usage.html#ssl-warnings
  warnings.warn(
11-18-2022 17:07:18 INFO     Collected commit_timestamp{ namespace=mongo-persistent, app=todolist, commit=247e9859fb232f2b59b17391c7402814d82651d9, image_sha=sha256:04559c4ebdde446ade9a67cd58124fdcb9e28f2ffc7fac923e0f3845c9f3d54a } 1668196791.0


```

### CODE REVIEW

File: exporters/committime/collector_azure_devops.py [hunk 1]
```diff
@@ -28,14 +28,16 @@ def get_commit_time(self, metric: CommitMetric):
...
+            "metric.git_api %s" % (str(self.git_api))
+        )  # TODO: metric, not self
 
-        # Private or personal token
         # Fill in with your personal access token and org URL
+        # personal_access_token = 'YOURPAT'
+        # organization_url = 'https://dev.azure.com/YOURORG'
         personal_access_token = self.token
-        organization_url = self.git_api
+        organization_url = str(self.git_api)
```
Comment (KevinMGranger) [inline exporters/committime/collector_azure_devops.py:38]: This will make the organization_url `None` if it's not set up. Is that correct?

**Comment by mpryc** [comment]
@KevinMGranger all the tests passed, only one dco is not, but I think we could merge this if you are happy with implementation?

**Comment by weshayutin** [comment]
```
process_max_fds 1.048576e+06
# HELP commit_timestamp Commit timestamp
# TYPE commit_timestamp gauge
commit_timestamp{app="todolist",commit="247e9859fb232f2b59b17391c7402814d82651d9",image_sha="sha256:04559c4ebdde446ade9a67cd58124fdcb9e28f2ffc7fac923e0f3845c9f3d54a",namespace="mongo-persistent"} 1.668196791e+09
```

**Review by KevinMGranger — APPROVED** [review]
/lgtm

**Pull request merged**